### DISCUSSION TITLE
Creating new column in database

### DISCUSSION THREAD

**Discussion by Midnitte** [ramnes/notion-sdk-py#39] [Q&A] [answered]
Good afternoon,

I'm trying to create page with three columns, following some examples in the documentation (and from the wiki page) I've been able to do it for two of the columns, but creating the third I seem to be doing something wrong with the format.
```new_page_prop = {
    'Name': {'title': [{'text': {'content': f"Test Page"}}]},
    'Link': {'type': 'url', 'url': f"http://test.com"},
    'Type': {'type': 'rich_text', 'rich_text': [{'text': f'Comment'}]}
}
notion.pages.create(
    parent={'database_id': my_database['id']},
    properties=new_page_prop
)
```
The column is a "text" column.
![image](https://user-images.githubusercontent.com/49769850/119381914-5a85de80-bc8f-11eb-80f7-ff2931c420f9.png)

Error:
```raise APIResponseError(error.response, body)
notion_client.errors.APIResponseError: body failed validation. Fix one: body.properties.Type.rich_text[0].text should be an object, instead was `"Comment"`. body.properties.Type.rich_text[0].mention should be defined, instead was `undefined`. body.properties.Type.rich_text[0].equation should be defined, instead was `undefined`. body.properties.Type.rich_text[0].id should be defined, instead was `undefined`. body.properties.Type.rich_text[0].name should be defined, instead was `undefined`.
```

Any help would be appreciated :)

**Comment by guedes** [comment, marked as answer]
Hi!

The value of key 'text' should be an object and should contain a key 'content', see https://developers.notion.com/reference/rich-text#text-objects

**Reply by Midnitte** [reply]
Ah got it, thanks!
